app/graph/mutations/tipline_newsletter_mutations.rb

```diff
@@ -0,0 +1,46 @@
+module TiplineNewsletterMutations
+  fields = {
+    enabled: 'bool',
+    introduction: 'str',
+    language: 'str'
+  }
+
+  # Header
+  fields.merge!({
+    header_type: 'str',
+    header_overlay_text: 'str'
+  })
+
+  # Content
+  fields.merge!({
+    content_type: 'str'
+  })
+
+  # Dynamic newsletter: RSS Feed
+  fields.merge!({
+    rss_feed_url: 'str',
+    number_of_articles: 'int'
+  })
+
+  # Static newsletter: Articles
+  fields.merge!({
+    first_article: 'str',
+    second_article: 'str',
+    third_article: 'str'
+  })
+
+  # Footer
+  fields.merge!({
+    footer: 'str'
+  })
+
+  # Schedule
+  fields.merge!({
+    send_every: 'json',
+    send_on: 'str',
+    timezone: 'str',
+    time: 'str'
+  })
```

Comment:
I think it's easier to define all fields at same time instead of using `.merge!` same as other mutations

Reply:
Hum... for big mutations, don't you think that it's more readable that way?